policy/modules/kernel/devices.if

```diff
@@ -6861,6 +6861,43 @@ interface(`dev_rw_sgx_vepc',`
 	rw_chr_files_pattern($1, device_t, sgx_vepc_device_t)
 ')
 
+########################################
+## <summary>
+##      Allow read and write the sgx_enclave devices
+## </summary>
+## <param name="domain">
+##      <summary>
+##      Domain allowed access.
+##      </summary>
+## </param>
+#
+interface(`dev_rw_sgx_enclave',`
+        gen_require(`
+                type device_t, sgx_enclave_device_t;
+        ')
+
+        rw_chr_files_pattern($1, device_t, sgx_enclave_device_t)
+        allow $1 sgx_enclave_device_t:chr_file { map execute };
```

Comment:
Please create a new interface for execution or use the _rwx_ interface name, depending on how it will possibly be used by other domains.

Reply:
Yes, this should have been 'rwx' .